### PR TITLE
Unable to change user role

### PR DESCRIPTION
**Pull request by rm-yakovenko**
`$role->first()` makes a db query and fetches the first item in the `roles` table. And it's **Owner**.

### CODE REVIEW

**Comment by Bottelet** [comment]
@rm-yakovenko Can you please update the user controller test, to ensure you need to be "OWNER" before you can do an update

**Comment by rm-yakovenko** [comment]
@Bottelet Fixed one more issue related to role changing.

> Can you please update the user controller test, to ensure you need to be "OWNER" before you can do an update

I'm not sure that I got it right. Let me know if the provided test isn't ok.

File: tests/Unit/Controllers/User/UsersControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+use App\Models\Role;
+use App\Models\User;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class UsersControllerTest extends TestCase
+{
+//    use DatabaseTransactions, WithoutMiddleware;
```
Comment (Bottelet) [inline tests/Unit/Controllers/User/UsersControllerTest.php:13]: Use both of these

File: tests/Unit/Controllers/User/UsersControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    /**
+     * @test
+     * @link https://github.com/Bottelet/DaybydayCRM/pull/175
+     */
+    public function can_update_user_role()
+    {
+        /** @var User $user */
+        $user = factory(User::class)->create();
```
Comment (Bottelet) [inline tests/Unit/Controllers/User/UsersControllerTest.php:22]: No reason to create a new user, just use `$this->user`

File: tests/Unit/Controllers/User/UsersControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        self::assertEquals(
+            [$targetRole->id],
+            $user->roles()->get()->pluck('id')->toArray()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function only_admin_can_update_user()
```
Comment (Bottelet) [inline tests/Unit/Controllers/User/UsersControllerTest.php:49]: It's everyone with the owner role

**Comment by Bottelet** [comment]
Thanks for your contribution! 
I made a few changes, but I really appreciate the work!

You can see the changes here if interested:
https://github.com/Bottelet/DaybydayCRM/commit/bff89c530600fe1c0fac56f5b488545726c29dbb

**Pull request merged**